nl/tested-judge/dsl/README.md

```diff
@@ -948,15 +968,24 @@ voorlopig niet ondersteund.
 Een **constructor** is een functieoproep die voorafgegaan wordt door het 
 sleutelwoord `new`.
 
-```javascript
+```tested
 new Counter()
 new object.Pair("Paar", 8.4e-5)
 ```
 
+### Globale variabelen
+
+Een **globale variabele** is een eigenschap van de globale namespace.
+Een **globale variabele** wordt genoteerd als een naam genoteerd tussen scheve haken.
```

Comment:
According to Wikipedia, these are called "punthaakjes, hoekankers, hoekhaakjes", but unsure what the proper term for these is in Dutch.

Reply:
"Scheve haken" is a proper term.
See page 29 on https://inigem.ugent.be/jvlfx/jvlfx.pdf.

Reply:
I don't know where prof. Coolsaet found this term, but I did some research, and I can't find a lot of usage of "scheve haken" for "<" ">". Vandale uses "punthaakjes" or "vishaakjes", so I would suggest taking one of those terms.

Reply:
I changed it in "punthaakjes".